package.json

```diff
@@ -15,7 +15,8 @@
     "react-places-autocomplete": "^5.4.3",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.0.14",
-    "react-social-login-buttons": "^1.3.5"
+    "react-social-login-buttons": "^1.3.5",
+    "speedtest-net": "^1.3.2"
```

Comment:
Did you want to add the implementation of the speedtest library to the app as well?

Reply:
Nope, it is not required here.